### DISCUSSION TITLE
Issue status

### DISCUSSION THREAD

**Discussion by gleb-svitelskiy** [Laravel-Backpack/community-forum#672] [Q&A (Help)] [answered]
Can I get some feedback on the [issue](https://github.com/Laravel-Backpack/CRUD/issues/5307) I posted?
It was moved to a repository that I don't have access to.

**Comment by jcastroa87** [comment, marked as answer]
Hello @gleb-svitelskiy there is a PR submitted and is on our review to validate QA if everything goes perfect will be release soon, maybe next week.

Cheers.
